src/SprykerEco/Client/Payone/Zed/PayoneStub.php

```diff
@@ -158,4 +160,14 @@ public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): Consumer
     {
         return $this->zedStub->call('/payone/gateway/send-consumer-score-request', $quoteTransfer);
     }
+
+    /**
```

Comment:
* Add `@uses` document block with a reference to the controller action.

Reply:
Thanks